projects/schematics/src/migrations/2_0/constructor-deprecations/data/cart-item-list.component.migration.ts

```diff
@@ -87,4 +87,46 @@ export const CART_ITEM_LIST_COMPONENT_MIGRATIONS: ConstructorDeprecation[] = [
       },
     ],
   },
+  {
```

Comment:
@znikola would you mind evaluating this file? I'm unsure if the different migration scripts are what we should end up with?